examples/2d/compute_speed.py

```diff
@@ -25,13 +25,39 @@
 # Certain backends are also GPU-only, we we want to detect that before running
 # the benchmark.
 
-import kymatio.scattering2d.backend as backend
+backends = []
+
+try:
+    if torch.cuda.is_available():
+        from skcuda import cublas
+        import cupy
+        from kymatio.scattering2d.backend import torch_skcuda_backend
+        backends.append(torch_skcuda_backend)
+except:
+    pass
+
+try:
+    from kymatio.scattering2d.backend import torch_backend
+    backends.append(torch_backend)
+except:
+    pass
+
+
+###############################################################################
+# Determine which devices (CPU or GPU) that are supported by the current
+# backend.
+
+if torch.cuda.is_available():
+    devices = ['cuda', 'cpu']
+else:
+    devices = ['cpu']
+
 
 ###############################################################################
-# Finally, we import the `Scattering2D` class that computes the scattering
+# Finally, we import the `Scattering2DTorch` class that computes the scattering
 # transform.
 
-from kymatio import Scattering2D
+from kymatio import Scattering2DTorch as Scattering2D
```

Comment:
So I'd like to see `Scattering2D` still existing, using dynamic inheritance.

Reply:
and a `logging.info` message: "Using X backend". Like in Keras, except that this one would be easy to turn off by raising the `LOGGING_LEVEL` (Keras uses `print` !)